subprojects/emcee/worker/src/main/kotlin/com/avito/emcee/worker/internal/identifier/HostnameWorkerIdProvider.kt

```diff
@@ -0,0 +1,10 @@
+package com.avito.emcee.worker.internal.identifier
+
+import java.net.InetAddress
+
+internal class HostnameWorkerIdProvider : WorkerIdProvider {
+
+    override fun provide(): String {
+        return InetAddress.getLocalHost().hostName
```

Comment:
Are you sure that host name will be different through the workers?

Reply:
yes, I checked

Reply:
I hope it wont break. Now it looks fragile. Maybe we could make a fail-fast check for that some how